### PR TITLE
Example Gallery

### PR DESCRIPTION
**Pull request by banesullivan**
These changes utilize `sphinx-gallery` to create a gallery page in the docs for all the examples:

ezgif com-video-to-gif-2

This allows users to easily download the examples as Jupyter notebooks to get up and running with `vtki`:

Screen Shot 2019-04-03 at 9 13 29 AM

See notes in https://github.com/sphinx-gallery/sphinx-gallery/issues/469 on how this was implemented

### Todo:

- [ ] Redeploy documentation via Travis - ReadtheDocs can't handle setting up all the dependencies to actually run `vtki` so we'll have to deploy the documentation from Travis from now on
- [x] Remove the annoying "Figure X" labels
- [x] Fix `make doctest`

### CODE REVIEW

**Comment by akaszynski** [comment]
This looks great!  Having a gallery is an excellent way of organizing the examples as figures.

**Comment by banesullivan** [comment]
Merging because documentation can only be deployed off the `master` branch. When I merge, this will break ReadtheDocs and I will have to rapidly switch over all the docs hosting. This will require a bit of debugging likely so the master branch might get messy for a minute

**Comment by banesullivan** [comment]
New docs are live on the go-pages branch. Waiting for domain records to refresh...

Note we’ll need to add a check in the Travis script to not deploy the docs unless on the master branch

**Comment by banesullivan** [comment]
Once everything looks good, I’m going to delete the Readthedocs project. @akaszynski, is that alright?

**Comment by banesullivan** [comment]
Resolve #134

File: vtki/plotting.py [hunk 1]
```diff
@@ -2743,7 +2764,6 @@ def show(self, title=None, window_size=None, interactive=True,
         # Render
         log.debug('Rendering')
         self.ren_win.Render()
-        img = self.screenshot(screenshot, return_img=True)
```
Comment (banesullivan) [inline vtki/plotting.py]: This deletion caused #186

**Pull request merged**